internal/pkg/model/orchestrator.go

```diff
@@ -25,13 +25,17 @@ type Phase struct {
 	Content   *orderedmap.OrderedMap `validate:"required"`
 }
 
+// Task is a part of an orchestration Phase.
+// Configuration can be defined by ConfigId or ConfigData.
+// ConfigId and ConfigData can be empty if Enabled=false.
 type Task struct {
 	TaskKey
 	AbsPath     `validate:"dive"`
 	Name        string `validate:"required"`
 	Enabled     bool
 	ComponentId storageapi.ComponentID `validate:"required"`
 	ConfigId    storageapi.ConfigID
+	ConfigData  *orderedmap.OrderedMap
```

Comment:
Pridal som podporu pre `configData` field, v tom pripade nie je `configId` resp. lokalne `configPath` definovana.
Skruktura commitov je rovnaka ako v predchadzajucom PR, musi sa to pridat do:
- parser (tento commit)
- local load
- local save
- remote load
- remote save